### DISCUSSION TITLE
Column description setting

### DISCUSSION THREAD

**Discussion by vishwa-ims** [RevealBi/Reveal.Sdk#148] [ASP.NET Core]
Hi,
We would like to know if we can set custom description for columns that are displayed under Data while editing or creating dashboards ? 

<img width="409" alt="image" src="https://github.com/RevealBi/Reveal.Sdk/assets/89643622/0323c369-3f6f-466e-8460-dbcd9ea77c99">

**Comment by vishwa-ims** [comment]
Hi Team
Do you have any updates please ?

**Reply by brianlagunas** [reply]
I apologize for the late response. No, you cannot provide a custom description for columns. That would be a feature request.
